crates/wasm-compose/example/server/Cargo.toml

```diff
@@ -9,7 +9,8 @@ async-std = { version = "1.12.0", features = ["attributes"] }
 clap = { version = "3.2.23", features = ["derive"] }
 driftwood = "0.0.6"
 tide = "0.16.0"
-wasmtime = { version = "6.0.0", features = ["component-model"] }
-wasmtime-component-macro = "6.0.0"
+wasi-cap-std-sync = { git = "https://github.com/bytecodealliance/preview2-prototyping" }
```

Comment:
If we want, we can pin this and `wasi-host` to `0d5ca07` to match that we document which `cargo-component` version we tested with.